yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/ActivityDetailsFactoryTest.java

```diff
@@ -126,6 +127,26 @@ public void shouldFailWithInvalidTimestamp() {
         fail("Expected an Exception");
     }
 
+    @Test
+    public void shouldFailWithInvalidExtraData() throws Exception {
+        Receipt receipt = new Receipt.Builder()
+                .withWrappedReceiptKey(validReceiptKey)
+                .withExtraData(EXTRA_DATA_CONTENT)
+                .build();
+
+        ExtraDataException extraDataException = new ExtraDataException("Failed to load extra data from receipt");
+        when(extraDataConverterMock.read(eq(EXTRA_DATA_CONTENT))).thenThrow(extraDataException);
+
+        try {
+            testObj.create(receipt, keyPair.getPrivate());
+        } catch (ProfileException e) {
+            assertThat(e.getMessage(), containsString("Failed to load extra data from receipt"));
+            assertTrue(e.getCause() instanceof ExtraDataException);
+            return;
+        }
+        fail("Expected an Exception");
```

Comment:
Is this exception being caught above this? We want to make sure that an exception with ExtraData isn't affecting the rest of the receipt parsing, so ideally we'd check that this error is thrown, logged, and the parsing continues.

Reply:
It is currently caught above, but we could change this to just catch it and log it (not affecting the rest of the parsing)

Reply:
Fixed in https://github.com/getyoti/yoti-java-sdk/pull/137/commits/aaa16277bd2bf529518c5465a2f8ade07a78c641